### ISSUE TITLE
INSERT INTO "acme_srv_housekeeping" -> value too long for type character varying(15)

### ISSUE THREAD

**Issue by rafaelorafaelo** [grindsa/acme2certifier#134]
Hi,
when I access MYURL/acme/renewal-info, I get a 500 error response in my browser. My Docker container uses a PostgreSQL database, where I find this error message:
```
value too long for type character varying(15)
INSERT INTO "acme_srv_housekeeping" ("name", "value", "modified_at") VALUES (E'cert_aki_serial_update', E'True', '2024-02-19T11:13:00.526431+00:00'::timestamptz) RETURNING "acme_srv_housekeeping"."id"
```
The table is defined as
```
CREATE TABLE IF NOT EXISTS public.acme_srv_housekeeping
(
    id integer NOT NULL DEFAULT nextval('acme_srv_housekeeping_id_seq'::regclass),
    name character varying(15) COLLATE pg_catalog."default" NOT NULL,
    value character varying(30) COLLATE pg_catalog."default" NOT NULL,
    modified_at timestamp with time zone,
    CONSTRAINT acme_srv_housekeeping_pkey PRIMARY KEY (id),
    CONSTRAINT acme_srv_housekeeping_name_key UNIQUE (name)
)
```
The string cert_aki_serial_update has more than 15 characters, which is the problem.
Regards,
Ralf

**Comment by grindsa** [comment]
Hi, 

Thx for your report. an Updated db-scheme including some other smaller fixes for better exception handling got included in [v0.33.2](https://github.com/grindsa/acme2certifier/releases/tag/0.33.2). Feel free to test and let me know it solves your problem.

BR G.

**Comment by rafaelorafaelo** [comment]
No more errors. Thanks.
